### PR TITLE
Updates to Exawind packages

### PR DESCRIPTION
**Pull request by psakievich**
These are a bunch of updates for packages in the Exawind software stack. @jrood-nrel @eugeneswalker

### CODE REVIEW

**Review by jrood-nrel — DISMISSED** [review]
I helped make a lot of these changes so I approve.

**Review by jrood-nrel — DISMISSED** [review]

**Review by eugeneswalker — DISMISSED** [review]
Thanks!

**Review by eugeneswalker — DISMISSED** [review]

File: var/spack/repos/builtin/packages/amr-wind/package.py [hunk 1]
```diff
@@ -95,6 +91,13 @@ def cmake_args(self):
             self.define_from_variant('AMR_WIND_TEST_WITH_FCOMPARE', 'tests'),
         ]
 
+        if '+mpi' in self.spec:
```
Comment (jrood-nrel) [inline var/spack/repos/builtin/packages/amr-wind/package.py]: We actually need to delete this block. This causes problems on Summit.

Comment (psakievich) [reply var/spack/repos/builtin/packages/amr-wind/package.py]: Done

**Review by jrood-nrel — DISMISSED** [review]

**Review by eugeneswalker — DISMISSED** [review]

**Review by jrood-nrel — DISMISSED** [review]

**Review by eugeneswalker — DISMISSED** [review]

**Review by jsitaraman — DISMISSED** [review]

File: var/spack/repos/builtin/packages/nccmp/package.py [hunk 2]
```diff
@@ -2,28 +2,20 @@
...
-        env.pop('F90', None)
-        env.pop('F90FLAGS', None)
-
-        configure('--prefix=%s' % prefix)
-        make()
-        make("check")
-        make("install")
+    def cmake_args(self):
+        args = []
+        return args
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/nccmp/package.py]: This can be removed

**Review by alalazo — CHANGES_REQUESTED** [review]
Only a very minor request.

File: var/spack/repos/builtin/packages/nccmp/package.py (hunk 2 above)
Comment (psakievich) [reply var/spack/repos/builtin/packages/nccmp/package.py]: Ah yeah. 🤦‍♂️ thanks!

**Comment by psakievich** [comment]
@alalazo requested changes made

**Review by alalazo — APPROVED** [review]

**Pull request merged**